dev-libs/pigpio/Manifest

```diff
@@ -0,0 +1 @@
+DIST V66.tar.gz 368677 BLAKE2B 9336dea1d4e8bb01d5ec25944b04fa38ef5a98f066b70e405258686628e8b4a6c0b0f989f7c5be38a728e2a842a5a6bfae43ea0c9d922ee5ae05d57e28b5bebd SHA512 f0c1ee4e3e394551f4c187ea2f4fae1e996e70c76de2f5b72baf0c1e96768288b1a4b8dd09cd87854fdb3b319dd57e9b205ae56c55b63491b89560138cbbe6b1
```

Comment:
the Manifest is wrong

Reply:
How do i generate it then? What should be in there? This was created by ebuild pigpio-66.ebuild digest

Reply:
fix the `SRC_URI` as I also pointed